content/en/dashboards/sharing.md

```diff
@@ -19,7 +19,40 @@ further_reading:
 
 ## Overview
 
-Shared graphs and screenboards allow you to display metric, trace, and log visualizations outside of Datadog.
+Shared dashboards and graphs allow you to display metric, trace, and log visualizations outside of Datadog.
+
+## Dashboards
+When you share a dashboard by URL or email link, the shared page shows live, read-only contents of that dashboard. 
+
```

Comment:
```suggestion

If your shared dashboard has visible tags configured, [template variable][4] selectors are present on the shared page. The template variables' default values are the default values you set when you configure the share, as described next.

```

Reply:
I'd rather keep this at the bottom and not lead with it

Reply:
I'd suggest we don't include this